platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/maps/MapboxMapOptions.java

```diff
@@ -912,7 +935,7 @@ public void writeToParcel(Parcel dest, int flags) {
     dest.writeByte((byte) (zoomGesturesEnabled ? 1 : 0));
     dest.writeByte((byte) (doubleTapGesturesEnabled ? 1 : 0));
 
-    dest.writeString(style);
+    dest.writeString(styleUrl);
     dest.writeString(apiBaseUrl);
```

Comment:
We are missing writing the `styleJson` to the parcel.

Reply:
nice catch 😅 fixed in https://github.com/mapbox/mapbox-gl-native/pull/12664/commits/fe04abb93deed836d9ff3ebe9850416ff0d4514c